src/opnsense/mvc/app/views/OPNsense/IPsec/key_pairs.volt

```diff
@@ -0,0 +1,123 @@
+{#
+Copyright (C) 2019 Pascal Mathis <mail@pascalmathis.com>
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+1. Redistributions of source code must retain the above copyright notice,
+   this list of conditions and the following disclaimer.
+
+2. Redistributions in binary form must reproduce the above copyright
+   notice, this list of conditions and the following disclaimer in the
+   documentation and/or other materials provided with the distribution.
+
+THIS SOFTWARE IS PROVIDED ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES,
+INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY
+AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY,
+OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+POSSIBILITY OF SUCH DAMAGE.
+#}
+
+<script>
+  $(document).ready(function () {
```

Comment:
```suggestion
  $(function () {
```

Reply:
Updated. This was a leftover from my first tries to get the UIBootgrid example from the docs working: https://wiki.opnsense.org/development/examples/using_grids.html#constructing-the-volt-template

Maybe those should be updated as well if `$(function() { ... })` is preferred over `$(document).ready(function() { ... })`?

Reply:
I think so. The others are deprecated according to the jQuery docs:

https://api.jquery.com/ready/

Just read the paragraph after the variants.

Reply:
CC @AdSchellevis